src/components/SiteHeader/SiteHeader.js

```diff
@@ -60,28 +60,28 @@ const SiteHeader = ({
 
               <HeaderNavigation aria-label="Main Navigation">
                 <HeaderMenu aria-label="Approach" className={approachClassName}>
-                  <HeaderMenuItem to="/approach" element={Link}>
+                  <HeaderMenuItem to="/approach/" element={Link}>
                     Overview
                   </HeaderMenuItem>
                   <HeaderMenuItem
                     to="/approach/design-philosophy"
```

Comment:
is there a reason you didn't give the same treatment to this link?

Reply:
nope, I missed it thanks